dspace/modules/xmlui-mirage2/src/main/webapp/themes/mit-fol/xsl/custom/core/page-structure.xsl

```diff
@@ -809,20 +809,21 @@
 
     </xsl:template>
 
-    <xsl:template name="addJavascript-google-analytics">
-        <!-- Add a google analytics script if the key is present -->
-        <xsl:if test="/dri:document/dri:meta/dri:pageMeta/dri:metadata[@element='google'][@qualifier='analytics']">
-            <script><xsl:text>
-                (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
-                (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
-                m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-                })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-
-                ga('create', '</xsl:text><xsl:value-of select="/dri:document/dri:meta/dri:pageMeta/dri:metadata[@element='google'][@qualifier='analytics']"/><xsl:text>', '</xsl:text><xsl:value-of select="/dri:document/dri:meta/dri:pageMeta/dri:metadata[@element='request'][@qualifier='serverName']"/><xsl:text>');
-                ga('send', 'pageview');
-            </xsl:text>
-            </script>
-        </xsl:if>
+    <xsl:template name="addJavascript-matomo-analytics">
```

Comment:
Looking at the change we made in DSpace@MIT for this, line 808 should change to reference `addJavascript-matomo-analytics ` instead of `addJavascript-google-analytics`

https://git.atmire.com/clients/mit-6/-/merge_requests/199/diffs

We also updated another xsl in the same changeset. Can you double check if we should do the same here?

Reply:
Yup, those template names should match. That was an oversight on my part. Good eye!